lib/streamlit/DeltaGenerator.py

```diff
@@ -1720,7 +1720,9 @@ def _check_and_convert_to_indices(options, default_values):
                 return None
 
             if not isinstance(default_values, list):
-                default_values = [default_values]
+                default_values = (
+                    list(default_values) if default_values else [default_values]
```

Comment:
Added the `if` in order to handle when `default_values=None`

Reply:
That inline `if` is a bit hard to read.

How about:
```
if default_values is None:
  default_values = [None]
else:
  default_values = list(default_values)
```

Reply:
Updated now. Became even more necessary as I now have `if/elif/else`! Thanks.